OMPython/__init__.py

```diff
@@ -860,36 +870,36 @@ def __init__(self, fileName=None, modelName=None, lmodel=[], useCorba=False, com
         self.setTempDirectory(customBuildDirectory)
 
         if fileName is not None:
-            self.loadLibrary(verbose)
-            self.loadFile(verbose)
+            self.loadLibrary()
+            self.loadFile()
 
         ## allow directly loading models from MSL without fileName
         if fileName is None and modelName is not None:
-            self.loadLibrary(verbose)
+            self.loadLibrary()
 
-        self.buildModel(variableFilter, verbose)
+        self.buildModel(variableFilter)
 
     def __del__(self):
         OMCSessionBase.__del__(self)
 
-    def setCommandLineOptions(self):
+    def setCommandLineOptions(self, commandLineOptions: str):
         ## set commandLineOptions if provided by users
         if commandLineOptions is not None:
-            exp="".join(["setCommandLineOptions(","\"",commandLineOptions,"\"",")"])
+            exp = "".join(["setCommandLineOptions(", "\"", commandLineOptions, "\"", ")"])
             cmdexp = self.getconn.sendExpression(exp)
             if not cmdexp:
-                return print(self.getconn.sendExpression("getErrorString()"))
+                self._check_error()
 
-    def loadFile(self, verbose):
+    def loadFile(self):
         # load file
-        loadFileExp="".join(["loadFile(","\"",self.fileName,"\"",")"]).replace("\\","/")
+        loadFileExp = "".join(["loadFile(", "\"", self.fileName, "\"", ")"]).replace("\\", "/")
         loadMsg = self.getconn.sendExpression(loadFileExp)
         ## Show notification or warnings to the user when verbose=True OR if some error occurred i.e., not result
-        if verbose or not loadMsg:
-            return print(self.getconn.sendExpression("getErrorString()"))
+        if not loadMsg:
```

Comment:
Forgot to add check for `self._verbose` flag.

Reply:
Done in an added commit; see below ...